### ISSUE TITLE
Headings in footer fail colour contrast 

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_base#591]
To replicate:

- localgov_base set as the default theme
- Add a block to the footer with the heading displayed to region: Footer first

Notice the heading is black ( #0b0c0c ) on the dark grey background. ( #505a5f ) 

![image](https://github.com/user-attachments/assets/4d7ca5df-e63b-47d9-8379-8b6588f30116)

I think this is 2.77:1 contrast ratio. 

For WCAG AA compliance
The visual presentation of regular text (14pt or 18.5px and below) and images of text has a contrast ratio of at least 4.5:1.
Large-scale text and images of large-scale text (14pt bold, 18pt or 24px and above) have a contrast ratio of at least 3:1.

https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

Maybe we need a variable for footer headers, defaulting to the `var(--color-pre-footer-text)`

**Comment by markconroy** [comment]
This is a little bit tricky. We have a `<footer>` element which wraps about 7 Drupal regions.

```
<footer>
  footer first | footer second | footer third
  footer
  lower footer first | lower footer second | lower footer third
</footer>
```

Some of those regions have a dark bg, and some have light. The "footer" region, for example, has white/no background colour set.

---
Thanks to [Big Blue Door](https://www.bigbluedoor.net/) for sponsoring my time to work on this.
